config/main.yml

```
@@ -71,6 +71,35 @@ roleGroups:
           Other Pronoun
           (please indicate in your nickname)
         emoji: '🇴'
+  - prompt: |-
+      Request Notifications (1/2)
+      Select reaction(s) on this message to be notified in DMs when specific requests are resolved. 
+    channel: '648479533246316555'
+    message: #TODO
+    radio: false
+    roles:
+      - id: #TODO
+      desc: Be notified of your resolved requests that are resolved as ✅.
+      emoji: '✅'
+      - id: #TODO
+      desc: Be notified of your resolved requests that are not resolved as ✅.
+      emoji: '☑️'
+  - prompt: |-
+      Request Notifications (2/2)
+      Select a reaction on this message if you want to be notified in DMs when requests are resolved after a certain amount of time.
+      Default is no time requirement.
```

```suggestion
      Select a reaction on this message if you want to only be notified about requests that are at least __ days old when they are resolved.
      No reaction means there is no time requirement.
```
I think the prompt I wrote above might be more clear. The prompt should tell the reader that the unit of time is days, and I think "no reaction" is clearer than "default".

---

I just realized that the description of each role/emoji specifies the amount of time that each emoji means, so specifying the unit of time in the prompt isn't as necessary as I had originally thought.

---

I added the "no reaction" message you suggested